samples/templates/default-azuredeploy-docker.json

```diff
@@ -692,20 +693,6 @@
                 "[resourceId('Microsoft.DocumentDb/databaseAccounts', variables('serviceName'))]"
             ]
         },
-        {
-            "condition": "[equals(parameters('solutionType'),'FhirServerCosmosDB')]",
-            "type": "Microsoft.KeyVault/vaults/secrets",
```

Comment:
NIT: Do we even need the keyvault anymore? Could be a new story.

Reply:
SQL is still using the keyvault too, it is at least consisntent. Might merge this first so I can mitigate having localauth enabled